app/src/org/commcare/activities/SyncCapableCommCareActivity.java

```diff
@@ -35,16 +36,40 @@
     private static final boolean SUCCESS = true;
     private static final boolean FAIL = false;
 
+    private static final String KEY_LAST_ICON_TRIGGER = "last-icon-trigger";
+    private static final String TRIGGER_ANIMATE_DATA_PULL = "animate-data-pull";
+    private static final String TRIGGER_ANIMATE_SEND_FORMS = "animate-send-forms";
+    private static final String TRIGGER_NO_ANIMATION = "no-animation";
```

Comment:
Did you change these to strings from an enum so that you can put them in the the bundle? I mention that because you can store enums in bundles: https://github.com/dimagi/commcare-android/blob/master/app/src/org/commcare/activities/UpdateUIController.java#L222